Readme.md

```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
 
 Also, you can observe the details of the implementation in the corresponding [demo project](https://git.haulmont.com/app-components/saml-addon-demo).
 
+## 4.2. Setup signing method for SAML messages <a name="setup-signing-method"></a>
+
+By default opensaml spring component uses sha1 digest algorithm for signing SAML messages. The most convenient way to use different signing message
```

Comment:
By default**,** **OpenSAML Spring** component uses **SHA1** digest algorithm for signing SAML messages. The most convenient way to use different signing message**s**

Reply:
fixed